spinalcordtoolbox/scripts/sct_label_vertebrae.py

```diff
@@ -418,13 +426,25 @@ def main(args=None):
              verbose=verbose,
              is_sct_binary=True,
              )
-    # Clean labeled segmentation
-    printv('\nClean labeled segmentation (correct interpolation errors)...', verbose)
-    clean_labeled_segmentation('segmentation_labeled.nii', 'segmentation.nii', 'segmentation_labeled.nii')
+
+    if clean_labels:
+        # Clean labeled segmentation
+        printv('\nClean labeled segmentation (correct interpolation errors)...', verbose)
+        clean_labeled_segmentation('segmentation_labeled.nii', 'segmentation.nii', 'segmentation_labeled.nii')
 
     # label discs
     printv('\nLabel discs...', verbose)
-    label_discs('segmentation_labeled.nii', verbose=verbose)
+    printv('\nUn-straighten labeled discs...', verbose)
+    run_proc('sct_apply_transfo -i %s -d %s -w %s -o %s -x %s' %
```

Comment:
same comment as in https://github.com/neuropoly/spinalcordtoolbox/pull/2990/files#r543841827, why not using the binary directly?

Reply:
nevermind again 😅 -- same reasoning: to be able to use the 'label' feature